test/suites/edgex-no-sec/status_test.go

```diff
@@ -20,8 +22,15 @@ func TestServiceStatus(t *testing.T) {
 
 func TestAccess(t *testing.T) {
 	t.Run("consul", func(t *testing.T) {
+		t.Log("Getting Consul token")
+		consulToken, _, _ := utils.Exec(t, "sudo cat /var/snap/edgexfoundry/current/secrets/consul-acl-token/bootstrap_token.json | jq -r '.SecretID'")
+
 		t.Log("Access Consul locally")
-		resp, err := http.Get("http://localhost:8500/v1/kv/edgex/core/2.0/core-data/Service/Port")
+		client := &http.Client{}
+		req, err := http.NewRequest("GET", "http://localhost:8500/v1/kv/edgex/v3/core-data/Service/Port", nil)
+		require.NoError(t, err)
+		req.Header.Set("X-Consul-Token", strings.TrimSpace(consulToken))
+		resp, err := client.Do(req)
```

Comment:
Passing the token in the request defeats the purpose of this test. Consul should not need a token when security is disabled. There may be a bug elsewhere.